### PR TITLE
Add circleci configuration

### PR DESCRIPTION
**Pull request by cagiti**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        chart: "harbor-config-operator"
+        requires:
+        - push-harbor-config-operator-to-quay
+        - push-harbor-config-operator-to-docker
+        filters:
+          # Trigger the job also on git tag.
+          tags:
+            only: /^v.*/
+
+# Disable collection updates temporarily to not break anything mistakenly
```
Comment (MarcelMue) [inline .circleci/config.yml:59]: Remove all this commented out stuff.

**Comment by MarcelMue** [comment]
You can exclude CVEs we can't do anything about with a `.nancyignore`:
example https://github.com/giantswarm/aws-operator/blob/master/.nancy-ignore

Please take a look though.

File: .circleci/config.yml (hunk 1 above)
Comment (cagiti) [reply .circleci/config.yml:59]: ok, will do

File: .nancyignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Fixed in https://github.com/beego/beego/releases/tag/v1.12.11 which is available in https://github.com/goharbor/harbor/releases/tag/v2.5.4. However, it appears that it's more likely that this PR (https://github.com/goharbor/harbor-operator/pull/970) will be merged first which also takes in beegov1.12.11.
```
Comment (MarcelMue) [inline .nancyignore:1]: Filename has to be `.nancy-ignore` I think

Comment (cagiti) [reply .nancyignore:1]: you're right, will update 👍

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**